src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CaseDataLiP.java

```diff
@@ -19,7 +21,7 @@ public class CaseDataLiP {
     private RespondentLiPResponse respondent1LiPResponse;
     @JsonProperty("applicant1LiPResponse")
     private ClaimantLiPResponse applicant1LiPResponse;
-    private TranslatedDocument translatedDocument;
+    private List<Element<TranslatedDocument>> translatedDocument;
```

Comment:
translatedDocuments - so it is clear there could be multiple documents

Reply:
updated name